.github/workflows/cut-release.yaml

```diff
@@ -0,0 +1,80 @@
+name: Cut Mod Release ⭐
+
+on:
+  workflow_dispatch:
+    inputs:
+      bump:
+        description: 'Semver Bump Type'
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+      binary_source:
+        description: 'Binary Source'
+        required: true
+        default: 'modbase'
+        type: choice
+        options:
+          - vanilla
+          - modbase
+          - build_binaries
+      jak1:
+        description: 'Supports Jak 1?'
+        required: true
+        default: 'true'
+        type: boolean
+      jak2:
+        description: 'Supports Jak 2?'
+        required: true
+        default: 'false'
+        type: boolean
+      jak3:
+        description: 'Supports Jak 3?'
+        required: true
+        default: 'false'
+        type: boolean
+      jakx:
+        description: 'Supports Jak X?'
+        required: true
+        default: 'false'
+        type: boolean
+
+permissions:
+  contents: write
+
+jobs:
+  prep_vars:
+    name: "Prep Variables"
+    runs-on: ubuntu-latest
+    outputs:
+      supported_games: ${{ steps.prep-vars.outputs.GAMES }}
+      tooling_repo: ${{ steps.prep-vars.outputs.REPO }}
+    steps:
+      - name: Prep Variables
+        id: prep-vars
+        run: |
+          GAMES=
+          if [[ ${{ inputs.jak1 }} == "true" ]]; then GAMES+=jak1,; fi
+          if [[ ${{ inputs.jak2 }} == "true" ]]; then GAMES+=jak2,; fi
+          if [[ ${{ inputs.jak3 }} == "true" ]]; then GAMES+=jak3,; fi
+          if [[ ${{ inputs.jakx }} == "true" ]]; then GAMES+=jakx,; fi
+          GAMES=${GAMES%?}
+          echo "GAMES=$GAMES" >> $GITHUB_OUTPUT
+          REPO=OpenGOAL-Mods/OG-Mod-Base
+          if [[ ${{ inputs.binary_source == 'vanilla' }} ]]; then REPO=open-goal/jak-project; fi
```

Comment:
alternatively if you build binaries using the vanilla `release-pipeline` workflow on your LL modbase, you can reuse those binaries in compatible mods for quicker release builds

that's what we're doing here by defaulting to `OpenGOAL-Mods/OG-Mod-Base`, those binaries are reused for most of the releases supported in the vanilla launcher right now

(if you choose `build_binaries` then the REPO variable is effectively ignored)

Reply:
Ok, I get it, so if I include the binaries in this modbase, then users of my modbase who don't make c++ changes may get faster builds..... I will think about it.

Reply:
yeah, since you have c++ changes different from jakmods modbase (e.g. audio stuff) I think there could be incompatibilities with jakmods prebuilt binaries. so you might want to have some for your LL-modbase too